pkg/util/podstore/podstore_test.go

```diff
@@ -408,6 +440,20 @@ func Test_podIPIndexFunc(t *testing.T) {
 	}
 }
 
+func Test_noHostNetworkPod(t *testing.T) {
```

Comment:
Maybe add one more Pod without `hostNetwork`, then verify that the filtered Pod len result is 1?